webapp/src/app/services/sharedService.ts

```diff
@@ -1,6 +1,9 @@
 export interface SharedService {
     saveModSelect(mod: string): Promise<void>;
     getSelectedMod(): string;
+	
+	saveWrapEventEditorLinesSetting(wrap: boolean): void;
```

Comment:
I used `SharedService` because because that's what the other settings used, so I just thought that's how settings persistance worked.
As for where the wrap setting should be I feel like the best way to do it is probably to have something like a `SettingsService` instead of putting it in `GlobalEventsService`, that way we don't mix persistent settings with other things and there is already a place for eventual future settings.

Reply:
sounds good